src/components/welcome/MyAccounts/MultiAccountItem.tsx

```diff
@@ -111,20 +113,35 @@ const MultiAccountItem = ({ onLinkClick, multiSafeAccountItem, safeOverviews }:
                 {shortenAddress(address)}
               </Typography>
             </Typography>
-
             <Typography variant="body2" fontWeight="bold" textAlign="right" pr={4}>
               {totalFiatValue !== undefined ? (
                 <FiatValue value={totalFiatValue} />
               ) : (
                 <Skeleton variant="text" sx={{ ml: 'auto' }} />
               )}
             </Typography>
-
-            <MultiChainIcon />
+            <Tooltip
+              title={
+                <Box>
+                  <Typography fontSize="14px">Multichain account on:</Typography>
+                  {safes.map((safeItem) => (
+                    <Box p="4px 0px" key={safeItem.chainId}>
+                      <ChainIndicator chainId={safeItem.chainId} />
+                    </Box>
+                  ))}
+                </Box>
```

Comment:
Would be nice if we can move it to be a separated component outside the `MultiAccountItem`? So we just need to call the reference of the component inside the tooltip instead of re-creating the component every-time the parent has a re-render?